tests/tests_e3/conftest.py

```diff
@@ -1,15 +1,26 @@
 # type: ignore
+from __future__ import annotations
+
 import os
+from os.path import abspath, dirname, join as path_join, isfile
```

Comment:
I do not know if the package importer is able to read that, but I am not ...\
Three imports for two names ?